packages/main/test-integration/node/cache-content.test.ts

```diff
@@ -0,0 +1,266 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect, use } from "chai";
+import * as chaiAsPromised from "chai-as-promised";
+import { GoogleGenerativeAI } from "../..";
+import { GoogleAICacheManager } from "../../server"; //= require("@google/generative-ai/server");
+
+use(chaiAsPromised);
+
+/**
+ * Integration tests against live backend.
+ */
+
+describe("cacheContent", function () {
+  this.timeout(60e3);
+  this.slow(10e3);
+  let text: string = "";
+  const endpointConfiguration = { /* remove at launch */ };
+
+  for (let i = 0; i < 1500; i++) {
+    text += "Purple cats drink gatorade.";
+    text += i % 8 === 7 ? "\n" : " ";
+  }
+  it("createCache", async () => {
+    // cacheManager create
```

Comment:
We create several cached contents every time we run the integration tests, and we don't delete them. Could this ever cause the tests to break (e.g. usage limits)?

Reply:
They have a very short TTL.

Reply:
Oh, right!